### ISSUE TITLE
Sourcemap upload not working for `.mjs` files

### ISSUE THREAD

**Issue by memcorrupt** [getsentry/sentry-javascript-bundler-plugins#508]
### Environment

vite `5.1.6`
@sentry/vite-plugin `2.16.0`

### Steps to Reproduce

in vite config:
- chunkFileNames ending in `.mjs` and/or entryFileNames ending in `.mjs`
- sourcemap generation enabled
- sentry vite plugin added

### Expected Result

Source maps are uploaded.

### Actual Result

No source maps are uploaded.

```
[sentry-vite-plugin] Debug: No `sourcemaps.assets` option provided, falling back to uploading detected build artifacts.
[sentry-vite-plugin] Warning: Didn't find any matching sources for debug ID upload. Please check the `sourcemaps.assets` option.
```

**Comment by lforst** [comment]
Hi, this seems like a bug and I think where it is. Do you know, did you define an output directory in vite?

**Comment by lforst** [comment]
@memcorrupt Hi, we just released version 2.16.1 of the package. Can you try whether it fixes the issue for you?
